packages/block-template-html/package.json

```diff
@@ -22,15 +22,14 @@
     "dev": "echo \"Serving at http://localhost:63212\" && http-server -p 63212 ./dev",
     "fix:eslint": "eslint --fix .",
     "format": "prettier --write --ignore-unknown .",
+    "postinstall": "rimraf ./dev/src && lnk -f ./src ./dev",
```

Comment:
Yarn Berry does not support `prepare`: https://yarnpkg.com/advanced/lifecycle-scripts